components/learn/modules/ROOT/pages/deploy-and-interact.adoc

```diff
@@ -186,17 +186,147 @@ TIP: To learn more about `send-tx` and `call`, check out the https://docs.openze
+async function main() {
+  // Set up web3 object, connected to the local development network
+  const web3 = new Web3('http://localhost:8545');
+}
+
+main();
+----
+
+Here, we are initializing a new `web3` instance connecting to the local development network we started earlier, that is running on localhost port 8545. We can test if the connection works by asking something to the local node, such as the list of enabled accounts:
+
+[source,js]
+----
+// Set up web3 object, connected to the local development network
+const web3 = new Web3('http://localhost:8545');
+
+// Retrieve accounts from the local node
+const accounts = await web3.eth.getAccounts();
+console.log(accounts);
+----
+
+NOTE: We will not be repeating the boilerplate code on every snippet, but make sure to always code _inside_ the `main` function we defined above!
+
+Run the code above using `node`, and check that you are getting a list of available accounts in response. 
+
+[source,console]
+----
+$ node src/index.js 
+[ '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1',
+  '0xFFcf8FDEE72ac11b5c542428B35EEF5769C409f0',
+  ... ]
+----
+
+These accounts should match the ones you got when you ran `openzeppelin accounts` earlier. Now that we have our first code snippet for getting data out of a blockchain, let's start working with our contract.
+
+[[getting-a-contract-instance]]
+=== Getting a contract instance
+
+In order to interact with our deployed `Box` contract, we will create a new https://web3js.readthedocs.io/en/1.2.4/web3-eth-contract.html[web3 contract instance]. This is a javascript object that represents our contract in the blockchain.
+
+To create this web3 contract, we need two things:
+
+* The address where it is deployed
+* The ABI of the contract, which is the definition of the contract's public functions
+
+The address was returned by the OpenZeppelin CLI when we deployed the contract. As for the ABI, we can retrieve it from the compiled artifact in the `build/contracts` folder as shown below.
+
+[source,js]
+----
+// Set up web3 object, connected to the local development network
+const web3 = new Web3('http://localhost:8545');
+
+// Set up a web3 contract, representing our deployed Box instance
+const address = '0xCfEB869F69431e42cdB54A4F4f105C19C080A601';
+const abi = require('../build/contracts/Box.json').abi;
+const box = new web3.eth.Contract(abi, address);
+----
+
+NOTE: Make sure to replace the `address` with the one you got when deploying the contract, which may be different to the one shown here.
+
+We can now use this javascript object to interact with our contract.
+
+[[calling-the-contract]]
+=== Calling the Contract
+
+Let's start by displaying the current value of the `Box` contract. We will need to https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#methods-mymethod-call[call] into the `retrieve()` public method of the contract, and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await[await] the response:
```

Comment:
Note that we've already used async/await syntax: we should link to the MDN docs then.

Reply:
Added